src/Indexing/Indexing.jsx

```diff
@@ -228,11 +228,13 @@ class Indexing extends React.Component {
         thisPointer.retrieveJobOutput(uid).then((resp) => {
           if (resp.data && resp.data.output) {
             const logsLink = resp.data.output.split(' ')[0];
+            const manifestLink = resp.data.output.split(' ')[1];
             thisPointer.setState({
               indexingFilesStatus: 'success',
               indexingFilesStatusLastUpdated: thisPointer.getCurrentTime(),
               indexingFilesPopupMessage: 'Done',
               indexingFilesLogsLink: logsLink,
+              indexingFilesManifestLink: manifestLink,
```

Comment:
does this state got set anywhere in the code?

Reply:
It is not in the constructor no -- it's only set here on this line you commented. Do you want it to be initialized to the empty string in the constructor, is that why you ask?

Reply:
cool!